src/main/java/com/ldtteam/structurize/client/gui/WindowShapeTool.java

```diff
@@ -193,8 +194,7 @@ else if (pos != null)
             genShape();
         }
 
-        
-        //final Button replaceButton = findPaneOfTypeByID(BUTTON_HOLLOW, Button.class);
+        final Button replaceButton = findPaneOfTypeByID(BUTTON_HOLLOW, Button.class);
```

Comment:
Why did you uncomment it when it's not used and was prepared for removal

Reply:
Why isn't it needed?

Reply:
Since it's local? and the line below use findPane again

Reply:
removed